src/styles/TodoListApp.styled.js

```diff
@@ -0,0 +1,144 @@
+import styled, { css } from "styled-components";
+import checkedIcon from "../../public/checked.png";
+import checkedIconGreen from "../../public/checked-green.png";
+import edit from "../../public/edit.png";
+import editChecked from "../../public/edit-checked.png";
+
+// ! INPUTS STYLING
+export const TodoInputAdd = styled.input.attrs(({ placeholder, type, value }) => {
+	placeholder: placeholder;
```

Comment:
Jeśli masz parametr o tej samej nazwie to możesz pominąć zapis typu
`{ placeholder: placeholder }` i zastąpić go `{ placeholder }`

Reply:
Tego nie wiedziałem, dzięki!!! 😃